.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+    - "master"
+  pull_request:
+    branches:
+    - "master"
```

Comment:
What is the idea behind only running the CI on the master branch?

Reply:
E.g. the actions in your fork had `*` in here. So it's harder to verify they work here, because the commits are different.

Reply:
I had "*" to test that the configuration works. I think it is quite common to have the actions configured only for the master branch and PRs targeting the master branch.